build/_build/Build.cs

```diff
@@ -53,6 +53,9 @@ partial class Build : NukeBuild
     [Parameter("The build version (for packaging purposes). Default is latest")]
```

Comment:
Version is no longer used only for "for packaging purposes" anymore, right?
```suggestion
    [Parameter("The build version. Default is latest")]
```